robots/FrankaEmikaPandaRobot.m

```diff
@@ -0,0 +1,55 @@
+% panda = FrankaEmikaPandaRobot.kinematics() returns a DQ_kinematics object
+% using the modified Denavit-Hartenberg parameters of the Franka Emika
+% Panda robot
+
+% (C) Copyright 2015-2022 DQ Robotics Developers
```

Comment:
The date range is incorrect. It should be 2011-2023

Reply:
Fixed in commit [81fe84a](https://github.com/dqrobotics/matlab/pull/102/commits/81fe84a53a0139e36040c25f8f38e4893855bd16).